templates/navigation_rustdoc.hbs

```diff
@@ -93,7 +93,7 @@
               <a href="/crate/{{name}}/{{version}}/source/" title="Browse source of {{name}}-{{version}}" class="pure-menu-link{{#if ../../varsb.package_source_tab}} pure-menu-active{{/if}}"><i class="fa fa-fw fa-folder-open-o"></i><span class="title"> Source</span></a>
             </li>
             <li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover">
-              <a href="#" class="pure-menu-link"><i class="fa fa-fw fa-gears"></i><span class="title"> Platform</span></a>
+              <a href="#" class="pure-menu-link" aria-label="Platform"><i class="fa fa-fw fa-gears"></i><span class="title"> Platform</span></a>
```

Comment:
What about "Platforms menu"?

Reply:
Or even "Platforms dropdown"?

Reply:
I should check to make sure this unordered list has an ARIA landmark at the top. If so, then a screen reader will know it's a list of menu items by context (yes, `role="navigation"` is a recommended best practice).

As for adding "dropdown" or "menu" to the descriptive text, that would be redundant. IMO, it's simply a menu to them: how we've chosen to represent it as sighted individuals couldn't matter less.